pages/partials/navTabs/administrator.ejs

```diff
@@ -0,0 +1,25 @@
+<%- include('navTab', {
+    activeSubPage:  'administrators',
+    urlSuffix:      '/administrator/admins',
+    iconClasses:    'fas fa-user-shield',
+    tabLabel:       'Administrators'}); %>
+<%- include('navTab', {
+    activeSubPage:  'server',
+    urlSuffix:      '/administrator/server',
+    iconClasses:    'fas fa-server',
+    tabLabel:       'Server'}); %>
```

Comment:
Could we rename the "Server" tab to be "Settings", to match the other PL pages that contain setting/config information (e.g., `pages/instructorCourseAdminSettings` and `instructorAssessmentSettings`).

Also, the convention we were using on the other pages is to put the tabs in alphabetical order. I'm not sure alphabetical is the best order, but at least it makes it possible to find things 😄 --- the tab order here is almost alphabetical, except that this one tab needs to move.